input/write_reactors.py

```diff
@@ -23,69 +25,123 @@ def write_reactors(csv_file, reactor_template, region_template, reactor_output,
 
     """
 
-    # display usage if in error
-    if len(sys.argv) <3:
-        print("Usage: 'Python write_reactors.py [csv] [reactor_template] [region_template]\
-             [reactor_output] [region_output]")
 
-    reactor_lists= np.genfromtxt(csv_file,
-                                delimiter=',',
-                                dtype=('S128','S128', 'int','int', 'int'),
-                                names=('country','reactor_name', 'capacity',
-                                       'n_assem_core','n_assem_batch'))
+def delete_file(file):
+    """
+    This function checks if there is a file with the same name,
+    and deletes the file if it exists.
+    """
+    if os.path.exists('./' + file) is True:
+        os.system('rm ' + file)
+
+
+def read_csv(csv_file):
+    """
+    This function reads the csv file and returns the array.
+
+    input)
+        csv_file: csv file that lists country, reactor name,
+                 capacity, #assem per core, # assem per batch.
+
+    return)
+        reactor_lists: array with the data from csv file
+    """
+
+    reactor_lists = np.genfromtxt(csv_file,
+                                 delimiter=',',
+                                 dtype=('S128', 'S128', 'int', 'int', 'int'),
+                                 names=('country', 'reactor_name', 'capacity',
+                                       'n_assem_core', 'n_assem_batch'))
+
+    return reactor_lists
+
+
+def read_template(template):
+    """
+    This function takes a jinja template file and returns it
+
+    input)
+        template: jinja template file
 
+    return)
+        jinja template object
+    """
     # takes second argument file as reactor template
-    with open(reactor_template, 'r') as fp:
+    with open(template, 'r') as fp:
         input_template = fp.read()
-        template = jinja2.Template(input_template) 
+        output_template = jinja2.Template(input_template)
+
+    return output_template
+
+
+def reactor_render(array, template, output_file):
+    """
+    This function takes the array and template and writes a reactor file
 
-    # takes third argument file as region template
-    with open(region_template,'r') as ft:
-        input_template2 = ft.read()
-        template2 = jinja2.Template(input_template2)
+    input)
```

Comment:
same comment about standardized format. If you make a habit of using a format that sphinx handles natively, you will easily be able to generate documentation websites automagically with sphinx in the long term future.